src/util/options.h

```diff
@@ -29,6 +29,8 @@ class optionst
   const value_listt &get_list_option(const std::string &option) const;
 
   void set_option(const std::string &option, const bool value);
+  //test
+  /// \param test
```

Comment:
Is this correct? There is no parameter called test in the below function declaration. Also, I'm not sure what the point of this documentation comment is, \param should have some text after the parameter describing what the parameter is fore.